### ISSUE TITLE
Cannot find module for ember-fullcalendar

### ISSUE THREAD

**Issue by iamareebjamal** [embroider-build/embroider#442]
After doing the first step listed in the README, embroider starts throwing error:

```
Build Error (WaitForTrees)

Cannot find module './vendor/fullcalendar/dist/fullcalendar.js' from '/tmp/embroider/606e06/node_modules/@embroider/synthesized-vendor'
```

Stacktrace:

```
  - broccoliBuilderErrorStack: Error: Cannot find module './vendor/fullcalendar/dist/fullcalendar.js' from '/tmp/embroider/606e06/node_modules/@embroider/synthesized-vendor'
    at Function.resolveSync [as sync] (/home/iamareebjamal/git/open-event-frontend/node_modules/resolve/lib/sync.js:89:15)
    at AppBuilder.impliedAddonAssets (/home/iamareebjamal/git/open-event-frontend/node_modules/@embroider/core/src/app.js:135:55)
    at AppBuilder.impliedAssets (/home/iamareebjamal/git/open-event-frontend/node_modules/@embroider/core/src/app.js:99:27)
    at AppBuilder.insertEmberApp (/home/iamareebjamal/git/open-event-frontend/node_modules/@embroider/core/src/app.js:207:36)
    at AppBuilder.prepareAsset (/home/iamareebjamal/git/open-event-frontend/node_modules/@embroider/core/src/app.js:333:18)
    at AppBuilder.prepareAssets (/home/iamareebjamal/git/open-event-frontend/node_modules/@embroider/core/src/app.js:343:18)
    at AppBuilder.updateAssets (/home/iamareebjamal/git/open-event-frontend/node_modules/@embroider/core/src/app.js:411:27)
    at AppBuilder.build (/home/iamareebjamal/git/open-event-frontend/node_modules/@embroider/core/src/app.js:505:38)
    at WaitForTrees.buildHook (/home/iamareebjamal/git/open-event-frontend/node_modules/@embroider/core/src/build-stage.js:37:31)
    at runMicrotasks (<anonymous>)
  - code: [undefined]
  - codeFrame: Cannot find module './vendor/fullcalendar/dist/fullcalendar.js' from '/tmp/embroider/606e06/node_modules/@embroider/synthesized-vendor'
  - errorMessage: Cannot find module './vendor/fullcalendar/dist/fullcalendar.js' from '/tmp/embroider/606e06/node_modules/@embroider/synthesized-vendor'
        at WaitForTrees (@embroider/compat/app)
-~- created here: -~-
    at new Plugin (/home/iamareebjamal/git/open-event-frontend/node_modules/@embroider/core/node_modules/broccoli-plugin/dist/index.js:45:31)
    at new WaitForTrees (/home/iamareebjamal/git/open-event-frontend/node_modules/@embroider/core/src/wait-for-trees.js:30:9)
    at CompatApp.get tree (/home/iamareebjamal/git/open-event-frontend/node_modules/@embroider/core/src/build-stage.js:26:16)
    at CompatApp.<anonymous> (/home/iamareebjamal/git/open-event-frontend/node_modules/typescript-memoize/dist/memoize-decorator.js:67:52)
    at new PackagerRunner (/home/iamareebjamal/git/open-event-frontend/node_modules/@embroider/core/src/to-broccoli-plugin.js:10:26)
    at Object.defaultPipeline [as compatBuild] (/home/iamareebjamal/git/open-event-frontend/node_modules/@embroider/compat/src/default-pipeline.js:35:12)
    at module.exports (/home/iamareebjamal/git/open-event-frontend/ember-cli-build.js:95:39)
    at Builder.readBuildFile (/home/iamareebjamal/git/open-event-frontend/node_modules/ember-cli/lib/models/builder.js:49:14)
    at Builder.setupBroccoliBuilder (/home/iamareebjamal/git/open-event-frontend/node_modules/ember-cli/lib/models/builder.js:63:22)
    at new Builder (/home/iamareebjamal/git/open-event-frontend/node_modules/ember-cli/lib/models/builder.js:29:10)
-~- (end) -~-
```

**Comment by iamareebjamal** [comment]
@ef4 ping

**Comment by iamareebjamal** [comment]
Hi @ef4, can I please get some help on the issue?

**Comment by NullVoxPopuli** [comment]
is full-calendar a component? or... what is it?

**Comment by GCheung55** [comment]
I'm guessing it's this addon: https://github.com/scoutforpets/ember-fullcalendar

**Comment by nightire** [comment]
I don't know too much about details, but it seems ember-fullcalendar uses old ember-cli-node-assets add-on. It is used to help developers to import dependencies from the `node_modules` directory IIRC. However, it's been superseded by ember-auto-import for a long time.

Perhaps this is the reason for the throwing error, and I suggest to contact the author(s) of ember-fullcalendar to see if they can help you to find the root cause. I think it's normal that a lot of ember add-ons doesn't work well with Embroider at the moment, I would not surprise that if ember-cli-node-assets doesn't work with Embroider for example.

**Comment by iamareebjamal** [comment]
I will raise a PR in the repo hopefully fixing the issue, thanks @nightire for indication on what may be the problem

**Comment by ef4** [comment]
I have seen the node-assets thing working correctly under embroider, so I’m
not sure if that’s it. I’ll need to take a look.

**Comment by iamareebjamal** [comment]
Still getting this error on latest embroider

```
Build Error (WaitForTrees)

Cannot find module './vendor/fullcalendar/dist/fullcalendar.js' from '/tmp/embroider/606e06/node_modules/@embroider/synthesized-vendor'
```

Judging from #563, I believe it is the same issue with `ember-cli-node-assets`. If creating a minimal reproduction repo will help, I'll do it as @ef4 has mentioned he can't reproduce the issue?

**Comment by ef4** [comment]
Fixed in #607
